dockers/docker-syncd-mlnx-rpc/Dockerfile

```diff
@@ -44,5 +44,6 @@ COPY supervisord.conf /etc/supervisor/conf.d/supervisord.conf
 COPY ptf_nn_agent.conf /etc/supervisor/conf.d/ptf_nn_agent.conf
 
 ENTRYPOINT service rsyslog start \
+    && sysctl -w net.core.rmem_max=509430500 \
```

Comment:
Because it makes sense and bound to the processes which lives inside of the container.